WcaOnRails/app/models/wfc_dues_redirect.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class WfcDuesRedirect < ApplicationRecord
+  belongs_to :redirect_to, class_name: "WfcXeroUser", foreign_key: "redirect_to_id"
```

Comment:
Based on my suggestion of using polymorphic types below (code comment in the migration file), you can add a relation here:

```suggestion
  belongs_to :redirect_to, class_name: "WfcXeroUser", foreign_key: "redirect_to_id"
  belongs_to :redirect_source, polymorphic: true
```